kissnet.hpp

```diff
@@ -909,9 +904,7 @@ namespace kissnet
 			if constexpr (sock_proto == protocol::tcp) //only TCP is a connected protocol
 			{
 
-				memcpy(&socket_output, getaddrinfo_results->ai_addr, sizeof(SOCKADDR));
-
-				int error = syscall_connect(sock, reinterpret_cast<SOCKADDR*>(&socket_output), sizeof(SOCKADDR));
+				int error = syscall_connect(sock, reinterpret_cast<SOCKADDR*>(getaddrinfo_results->ai_addr), socklen_t(getaddrinfo_results->ai_addrlen));
```

Comment:
The reinterpret_cast id not needed here.

Reply:
Done.